src/app/container/versions/versions.component.html

```diff
@@ -1,76 +1,78 @@
-<div class="container">
+<div>
   <div class="row">
     <div>
       <table datatable [dtOptions]="dtOptions" class="table row-border hover">
         <thead>
-          <tr>
```

Comment:
Looks like <tr> is no longer nested inside <thead> due to the whitespace changes